operator/v1/types.go

```diff
@@ -52,6 +52,33 @@ type ClusterManagerSpec struct {
 	// +optional
 	// +kubebuilder:default={mode: Default}
 	DeployOption DeployOption `json:"deployOption,omitempty"`
+
+	// Detached include configurations we needs in the detached mode.
+	// +optional
+	Detached Detached `json:"detached"`
+}
+
+// Detached represents customized configurations we need to set in the detached mode.
+type Detached struct {
```

Comment:
Maybe DetachedClusterManagerConfiguration for the struct name, you cannot ensure we do not have klusterlet configuration later.

Reply:
Sure. it's better to have this clearification.